feat: flex gap docs (0.71.0 release) (#3398)
facebook/react-native-website: docs/layout-props.md

---

### `columnGap`

> `columnGap` property sets the size of the gap (gutter) between an element's columns. It works like `column-gap` in CSS. See https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap for more details.

Reviewer: One difference compared to what we link to on MDN is that we don't support percentages. We should maybe specify explicitly that the number here is pixels.

---

### `columnGap`

> `columnGap` property sets the size of the gap (gutter) between an element's columns. It works similar to `column-gap` in CSS, but in React Native you must use points. Percentage unit is not supported.
> 
> See https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap for more details.

Reviewer: ```suggestion
`columnGap` works like `column-gap` in CSS. Only pixel units are supported in React Native. See https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap for more details.
```

Author: > pixels != points

oh, I thought points were RN's way of saying device-independent pixels. Is that not the case? e.g. https://reactnative.dev/docs/layout-props#width

Author: okay, just realized you mean `points` = pt unit. i.e. 1pt = 1/72nd of 1in 😅. yeah, indeed that's not the case. But not sure we can call it pixel units as its value doesn't end with `px` suffix. Although it behaves like CSS pixel though 😅 (device independent). Maybe I am overthinking. I'll just push the commit. We can discuss it somewhere else.

---

### `gap`

> `gap` property sets the size of the gap (gutter) between rows and columns. It is a shorthand for `rowGap` and `columnGap`. It works similar to `gap` in CSS, but in React Native you must use points. Percentage unit is not supported.
> 
> See https://developer.mozilla.org/en-US/docs/Web/CSS/gap for more details.

Reviewer: ```suggestion
`gap` works like `gap` in CSS. Only pixel units are supported in React Native. See https://developer.mozilla.org/en-US/docs/Web/CSS/gap for more details.
```

---

### `rowGap`

> `rowGap` property sets the size of the gap (gutter) between an element's rows. It works similar to `row-gap` in CSS, but in React Native you must use points. Percentage unit is not supported.
> 
> See https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap for more details.

Reviewer: ```suggestion
`rowGap` works like `row-gap` in CSS. Only pixel units are supported in React Native. See https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap for more details.
```
